test/e2e/step3.spec.ts

```diff
@@ -0,0 +1,1087 @@
+import { expect } from '@playwright/test'
+import { test, formStateFixture, websiteURL } from './util'
+
+test.use({
+  locale: 'nl-NL',
+  timezoneId: 'Europe/Amsterdam',
+  geolocation: { latitude: 51.6045656, longitude: 5.5342026 },
+})
+
+interface MyTextConfig {
+  name: string
+  testConfig: { colorScheme?: string }
+  forcedColors?: boolean
+}
+const parameters: MyTextConfig[] = [
+  {
+    name: 'Light mode',
+    testConfig: { colorScheme: 'light' },
+    forcedColors: false,
+  },
+]
+
+parameters.forEach(async ({ name, testConfig, forcedColors }) => {
+  test.describe(`${name}`, () => {
+    const pageURL = `${websiteURL}nl/incident/contact`
+    if (testConfig) {
+      test.use(testConfig as any)
+    }
+
+    test('has title', async ({ page, context }) => {
+      formStateFixture(context, {
+        description: 'lamp',
+        coordinates: [51.61892134, 5.52874105],
+        address: {
+          coordinates: [5.52874105, 51.61892134],
+          id: 'adr-987acc537500c2f62ab7449a6d1e6f2e',
+          postcode: '5462GJ',
+          huisnummer: '3A',
+          woonplaats: 'Veghel',
+          openbare_ruimte: 'Lage Landstraat',
+          weergave_naam: 'Lage Landstraat 3A, 5462GJ Veghel',
+        },
+        last_completed_step: 2,
```

Comment:
Kan het zijn dat `last_completed_step` verwijderd kan worden uit onze fixtures? Als het goed is had ik dingen zo gerefactored dat deze niet value niet meer nodig is.